docs/pdf-lesson-plan-generation.md

```diff
@@ -57,3 +57,10 @@ For Dropbox folder access, ask Brendan or anyone with Dropbox `Pegasus Staging`
   brew install https://raw.github.com/quantiverge/homebrew-binary/pdftk/pdftk.rb
   brew install enscript
   ```
+
+## To run PDF collation tests
+
+1. In `/Gemfile`, un-comment the line with `gem 'pdf-reader'`
+1. `bundle install`
+1. `cd code-dot-org/pegasus/test/disabled`
+1. `bundle exec ruby -Itest test_pdf_merger.rb`
```

Comment:
What do you think about moving `gem 'pdf-reader'` to `group :development, :test` in the Gemfile? It looks like that's what we do for other test dependencies.